### PR TITLE
Updating ui

### PR DESCRIPTION
**Pull request by wgarcia4190**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://www.pokt.network/wp-content/uploads/2020/06/cropped-pocket-icon-192x192.png" />
```
Comment (nymd) [inline public/index.html]: At first I used a WordPress linked logo for the repositories, and then someone changed WordPress and all the images broke. We should come up with a better way to host this image - - - IPFS? Or an S3 bucket.

I'll give you a link to the logo on S3 shortly

File: src/app.js [hunk 2]
```diff
@@ -36,6 +37,10 @@ class App extends Component {
 
   }
 
+  componentDidMount(){
+    document.title = "Block Explorer"
```
Comment (nymd) [inline src/app.js]: "Pocket Blockchain Block Explorer"

File: src/components/footer/index.js [hunk 3]
```diff
@@ -50,30 +50,26 @@ class Footer extends Component {
...
                 <li className="mainli">
                   <div className="title-sm">RESOURCES</div>
                   <ul className="sub-menu">
-                    <li className="menu-item"><a href="http://example.com">WhitePaper</a></li>
-                    <li className="menu-item"><a href="http://example.com">Economic One Pager</a></li>
-                    <li className="menu-item"><a href="http://example.com">iOS SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">Android SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">JS SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">Press kit</a></li>
+                    <li className="menu-item"><a href="https://pokt.network/wp-content/uploads/2020/03/POCKET-NETWORK-WHITEPAPER-V0.3.0.pdf">WhitePaper</a></li>
```
Comment (nymd) [inline src/components/footer/index.js]: These should be on S3 as well so they don't decay

File: src/datasource/datasource.js [hunk 4]
```diff
@@ -223,11 +226,11 @@ export class DataSource {
 
     async getBalance() {
         const pocket = await this.getPocketInstance()
-        const pocketAddress = config.address
+        const pocketAddress = "F7E8637E13033279095CFCC9E50548EC78E4C0FD";//config.address
```
Comment (nymd) [inline src/datasource/datasource.js]: This shouldn't be a hardcoded address should it?

File: src/views/account/details/index.js [hunk 5]
```diff
@@ -2,28 +2,53 @@ import React from 'react';
...
     }
 
     render() {
+
+        function AlertMessage() {
+            return (
+                <Alert variant="danger" dismissible>
+                    <Alert.Heading>NO RESULT FOR THIS SEARCH!</Alert.Heading>
+                    <p>
+                        Try searching by Transaction hash, Block number or Account address
```
Comment (nymd) [inline src/views/account/details/index.js]: Remove capital letter on transaction, block, and account

File: src/views/block/details/index.js [hunk 6]
```diff
@@ -43,8 +74,30 @@ class BlockDetails extends React.Component {
     }
 
     render() {
+
+        function AlertMessage() {
+            return (
+                <Alert variant="danger" dismissible>
+                    <Alert.Heading>NO RESULT FOR THIS SEARCH!</Alert.Heading>
+                    <p>
+                        Try searching by Transaction hash, Block number or Account address
```
Comment (nymd) [inline src/views/block/details/index.js]: Remove capital letters

File: src/views/block/details/index.js [hunk 7]
```diff
@@ -4,21 +4,52 @@ import Details from '../../../components/details';
...
+        this.dataSource.getLatestTransactions(1, 100, this.height).then(txs => {
+            if(txs.length !== 0) {
+                const latestArray = []
+                txs.forEach(tx => {
+                    console.log(tx)
+                    const latest = new LatestInfo(
+                        tx.height.toString(),
+                        tx.id,
+                        undefined,
+                        "POCKET TESTNET",
```
Comment (nymd) [inline src/views/block/details/index.js]: Should this be just "testnet"?

File: src/views/home/index.js [hunk 8]
```diff
@@ -30,18 +35,41 @@ class Home extends React.Component {
...
                     totalStakedApps={this.state.totalApps}
                 />
+                <div className="alert" style={this.state.showMessage ? {} : { display: 'none' }}>
+                    <img src={altertT} alt="alert" />
+                    <div className="cont-alert">
+                        <div className="title">
+                            <h3>NO RESULT FOR THIS SEARCH!</h3>
+                        </div>
+                        <p>
+                            Try searching by Transaction hash, Block number or Account address
```
Comment (nymd) [inline src/views/home/index.js]: Capitals

File: src/views/tx/details/index.js [hunk 9]
```diff
@@ -2,27 +2,59 @@ import React from 'react';
...
         this.dataSource.getTransaction(this.hash).then(tx => {
-            if(tx !== undefined)
-                this.setState({ txId: tx.id, txHash: tx.height.toString(), time: tx.data.index, network: "POCKET TESTNET", data: tx.data})
+            if(tx !== undefined) {
+                this.setState({
+                    txId: tx.id,
+                    txHash: tx.height.toString(),
+                    time: tx.data.index,
+                    network: "POCKET TESTNET",
+                    data: tx.data
```
Comment (nymd) [inline src/views/tx/details/index.js:25]: Testnet?

File: src/views/tx/details/index.js [hunk 10]
```diff
@@ -2,27 +2,59 @@ import React from 'react';
...
     }
 
     render() {
+
+        function AlertMessage() {
+            return (
+                <Alert variant="danger" dismissible>
+                    <Alert.Heading>NO RESULT FOR THIS SEARCH!</Alert.Heading>
+                    <p>
+                        Try searching by Transaction hash, Block number or Account address
```
Comment (nymd) [inline src/views/tx/details/index.js]: Capitals

**Review by nymd — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (wgarcia4190) [reply public/index.html]: Thanks

File: src/app.js (hunk 2 above)
Comment (wgarcia4190) [reply src/app.js]: got it

File: src/datasource/datasource.js (hunk 4 above)
Comment (wgarcia4190) [reply src/datasource/datasource.js]: nope, thanks

File: src/views/account/details/index.js (hunk 5 above)
Comment (wgarcia4190) [reply src/views/account/details/index.js]: Ok

File: public/index.html (hunk 1 above)
Comment (nymd) [reply public/index.html]: https://pocket-network-assets.s3-us-west-2.amazonaws.com/logos/cropped-pocket-icon-192x192.png

File: src/components/footer/index.js (hunk 3 above)
Comment (nymd) [reply src/components/footer/index.js]: https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Whitepaper-v0.3.0.pdf

Economic one pager needs a different link: 
https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Economic-Paper-v1.0.2.pdf

**Review by nymd — CHANGES_REQUESTED** [review]

**Review by nymd — APPROVED** [review]

**Pull request closed without merging**